src/SdfEntityCreator.cc

```diff
@@ -788,8 +793,33 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Visual *_visual)
   // \todo(louise) Populate with default material if undefined
   if (_visual->Material())
   {
+    sdf::Material visualMaterial = *_visual->Material();
+    if (!_visual->Material()->ScriptUri().empty())
+    {
+      std::string scriptUri = visualMaterial.ScriptUri();
+      if (scriptUri != "file://media/materials/scripts/gazebo.material") {
+        gzwarn << "Custom material scripts are not supported."
+          << std::endl;
+      }
+    }
+    if (!_visual->Material()->ScriptName().empty())
+    {
+      std::string scriptName = visualMaterial.ScriptName();
+
+      if((scriptName.find("Gazebo/") != std::string::npos)) {
```

Comment:
looks like the material name starts with this substring, so how about:

```suggestion
      if ((scriptName.find("Gazebo/") == 0u)) {
```